examples/manipulating_visualizing/plot_dim_plotting.py

```diff
@@ -0,0 +1,51 @@
+"""
+Controling the contrast of the background when plotting
+=========================================================
+
+The `dim` argument controls the contrast of the background.
+
+*dim* modifies the contrast of this image: dim=0 leaves the image
+unchanged, negative values of *dim* enhance it, and positive values
+decrease it (dim the background).
+"""
+
+# Retrieve the data: the localizer dataset with contrast maps
+
+from nilearn import datasets
+
+localizer_dataset = datasets.fetch_localizer_contrasts(
+    ["left vs right button press"],
+    n_subjects=2,
+    get_anats=True,
+    get_tmaps=True)
+localizer_anat_filename = localizer_dataset.anats[1]
+localizer_tmap_filename = localizer_dataset.tmaps[1]
+
+# Plotting: vary the 'dim' of the background
+from nilearn import plotting
+
+plotting.plot_stat_map(localizer_tmap_filename,
+                       bg_img=localizer_anat_filename,
+                       cut_coords=(36, -27, 66),
+                       threshold=3, title="dim=-.5",
+                       dim=-.5)
+
+plotting.plot_stat_map(localizer_tmap_filename,
+                       bg_img=localizer_anat_filename,
+                       cut_coords=(36, -27, 66),
+                       threshold=3, title="dim=0",
+                       dim=.0)
```

Comment:
> I think that '0' would be less surprising than '.0',

That's what I meant: 0 or 0. i.e with the dot before might be less confusing than .0 (dot after the 0)

For the record, this is a markdown gotcha (that I already bumped into recently actually). Starts a line with a 0. and it will be interpreted as a numbered list and replace by 1.